config/locales/en.yml

```diff
@@ -1167,10 +1178,26 @@ en:
       new:
         heading: Sign into your account
         forgot_password: "I've forgotten my password"
+      logged_out:
+        heading: You have signed out
+        lead_text: Use the link emailed to you to return to your saved case.
+        more_text_html: <a href="https://www.gov.uk/">Go to GOV.UK</a>
+      signin_save_confirmation:
+        heading: Your appeal has been saved
+        lead_text_html: A confirmation email has been sent to:<br><strong>%{user_email}</strong>
+        expire_warning: We will keep a draft of your appeal for %{expire_in_days} days. Any information will be automatically deleted for your security after this time.
```

Comment:
This is the same block of values, repeated twice. Can we move this to a YAML alias, please?